draft-ietf-deleg.md

```diff
@@ -264,61 +264,55 @@ Rest of the Step 2's description is not affected by this document.
 Please note the instructions to "Bound the amount of work" further down in the original text to apply. Suitable limits MUST be enforced to limit damage EVEN IF SOMEONE HAS INCORRECTLY CONFIGURED SOME DATA.
 
 ## Authoritative Servers
-DELEG aware authoritative servers have to act different when handling DE=0 queries from DELEG unaware clients, and DE=1 queries from DELEG aware clients.
+DELEG-aware authoritative servers act differently when handling queries from DELEG-unaware clients (those with DE=0) and queries from DELEG-aware clients (those with DE=1).
 
-### Unaware clients, DE=0
-A query with DE EDNS bit cleared indicates a DELEG unaware client. Such client has no use for DELEG information. In this case DELEG aware authoritative server MUST answer as if it is DELEG unaware.
+DNSSEC signing in the presence of DELEG records is unaffected.
```

Comment:
I find this confusing. Signing IS affected by presence of DELEG records, but the proofs we send back to clients are not affected by DE bit in the query.